Header enum_header
string type
string state
string success